actions/src/robocorp/actions/__init__.py

```diff
@@ -32,11 +33,17 @@ def enter_user() -> str:
         func: A function which is a action to `robocorp.actions`.
     """
 
-    # i.e.: This is just a thin layer for the task decorator at this point
-    # (it may be extended in the future...).
-    from robocorp.tasks import task
+    def decorator(func, **kwargs):
+        # i.e.: This is just a thin layer for the task decorator at this point
+        # (it may be extended in the future...).
+        from robocorp.tasks import task
 
-    return task(func)
```

Comment:
Question: can't you just `return task(*args, **kwargs)` here?

Reply:
Humm, this was marked as resolved, but I'm not sure it was done...

I meant for the full implementation of action to be:
```
def action(*args, **kwargs):
    return task(*args, **kwargs)
```

Doesn't that work for some reason?

Reply:
Don't we want to validate the `@action(...)` args?